app/frontend/actions.ts

```diff
@@ -841,30 +808,15 @@ export default ({setState, getState}: {setState: SetStateFn; getState: GetStateF
   const redeemRewards = async (state) => {
     loadingAction(state, 'Preparing transaction...')
     const rewards = state.shelleyBalances.rewardsAccountBalance
-    const balance = state.balance
-    let plan
-    try {
-      plan = await prepareTxPlan({
-        rewards,
-        txType: 'redeem',
-      })
-    } catch (e) {
-      setErrorState('transactionSubmissionError', e, {
-        shouldShowTransactionErrorModal: true,
-      })
-      return
-    }
-    // const cryptoProviderVersionError = wallet.checkCryptoProviderVersion()
+    const plan = await prepareTxPlan({rewards, txType: 'redeem'})
     const redemptionValidationError =
```

Comment:
Rename redemptionValidationError to withdrawalValidationError if you started renaming it in all other places to get rid of the term 'redeem' completely for consistency

Reply:
Good point, fixed.